tdrs-backend/tdpservice/reports/serializers.py

```diff
@@ -1,16 +1,21 @@
 """Serialize stt data."""
 
 from rest_framework import serializers
-from ..stts.models import STT
-from ..users.models import User
-from .models import ReportFile
 
-from .errors import ImmutabilityError
+from tdpservice.reports.errors import ImmutabilityError
```

Comment:
Partly a personal nit, but using absolute paths instead of relative paths here prevents a lot of issues that can occur as code gets moved around and additionally allows IDEs like PyCharm to handle some types of refactoring automatically.

Reply:
Agreed